loadtest/store/store.go

```diff
@@ -9,14 +9,14 @@ import (
 
 type UserStore interface {
 	Id() string
+	Post(postId string) (*model.Post, error)
+	User() (*model.User, error)
+	Channel(channelId string) (*model.Channel, error)
```

Comment:
This is important. There was a reason why those were in the `MutableUserStore`. We are returning a pointer to the actual data stored and in the case of the memory store that's also a way to alter that data. Whatever method ends up in the `UserStore` interface needs to return a copy otherwise we break immutability. I would keep this unchanged for now to avoid dealing with [deep copying](https://en.wikipedia.org/wiki/Object_copying#Deep_copy) until we really need it.

Reply:
@streamer45 
Understood. But:
How should `channel, err := c.user.Store().Channel("")` work now that there isn't a Channel() method in immutable store?

Reply:
The ticket was about implementing the `User` method. We lack an important piece still (comment above) before we can make the controller function work properly. I would't worry about this function too much. We'll get back to it in due time.